camelot/core/orm/__init__.py

```diff
@@ -90,7 +90,10 @@ def setup_all( create_tables=False, *args, **kwargs ):
     """
     if create_tables:
         metadata.create_all( *args, **kwargs )
-        
+
+# Assert that EntityMeta does not allow rebinding entity naming contexts.
+# This flag should only be enabled in test scenarios.
+assert EntityMeta.rebind == False
```

Comment:
alternatief id might be om de id(entity) mee in het pad te steken, zodat er meerdere classes met dezelfde naam kunnen gebind zijn, of een entity_args name voorzien die die uniek moet zijn of zo en default naar de class name

(tbd : mss ook in de context van de memento)

Reply:
Fixed in https://github.com/v-finance/camelot/pull/336/